pkg/construct2/graph_io.go

```diff
@@ -119,10 +124,20 @@ func (g *YamlGraph) UnmarshalYAML(n *yaml.Node) error {
 	}
 
 	var errs error
-	for rid, props := range y.Resouces {
+	for rid, props := range y.Resources {
+		var imported bool
+		if imp, ok := props["imported"]; ok {
+			val, ok := imp.(bool)
+			if !ok {
+				errs = errors.Join(errs, fmt.Errorf("unable to parse imported value as boolean for resource %s", rid))
+			}
+			imported = val
+			delete(props, "imported")
+		}
 		err := g.Graph.AddVertex(&Resource{
 			ID:         rid,
 			Properties: props,
+			Imported:   bool(imported),
```

Comment:
nit: why the cast, isn't imported already a bool?

Reply:
must have been an autocomplete that i missed, will remove